tests/test_id_block.py

```diff
@@ -7,12 +7,6 @@
 import base64
 from sevsnpmeasure import id_block
 
-# Generate test id_key
-# openssl genrsa -out keyfile/id_key_test.pem 2048
-
-# Generate test author_key
-# openssl genrsa -out keyfile/author_key_test.pem 2048
-
```

Comment:
I agree with @fitzthum 's suggestion.

@shuk777 - please restore the comments, but change the command-lines to the ones that actually generate the EC keys (such as those we have committed in `tests/keyfile/`).

Reply:
fixed